xblock/mixins.py

```diff
@@ -242,6 +249,13 @@ def _field_data(self, field_data):
         warnings.warn("Setting _field_data is deprecated", FieldDataDeprecationWarning, stacklevel=2)
         self._deprecated_per_instance_field_data = field_data
 
+    @property
+    def supports_save(self):
+        """
+        Returns true if this block supports being saved.
+        """
+        return True
```

Comment:
If I'm understanding this correctly, we're putting this here so that the runtime code can generically call render on XBlocks that want to save data after a view is executed, and UIBlocks that do not support that functionality. But if that's the case, maybe we could make the runtime not knowing about saving at all, but more generically execute post-rendering hooks? XBlock would then define post-rendering to do a `save()`, while UIBlocks would just do a no-op?